assets/src/edit-story/components/inspector/document/publish/utils.js

```diff
@@ -15,27 +15,35 @@
  */
 
 /**
- * External dependencies
+ * Adds 0 in front of the number if it's less than 10.
+ *
+ * @param {number} number Number to check.
+ * @return {string} Converted number.
  */
-import moment from 'moment';
+function ensureDoubleDigitNumber(number) {
+  // This could also be done via 0 + value and slicing, however, comparison is faster.
+  return number < 10 ? '0' + number : String(number);
+}
 
 export function getReadableDate(date, is12Hours = true) {
-  const displayDate = date ? moment(date) : moment();
+  const displayDate = date ? new Date(date) : new Date();
```

Comment:
Should we just use DateTimeFormat?

Reply:
Wouldn't this require using the locale? The display is currently based on a WordPress setting which can be changed by the user. We could perhaps discuss using the locale generally separately if it seems like we should use the browser locale instead. Thoughts?

Reply:
Ok. Didn't quire realize exactly what's customizable in WP settings. But what exactly does WP customize? Is there a way to derive a locale from WP settings? If so, we can certainly use standard formatting tools.

Reply:
Locale, date format, time format, and so on:

<img width="904" alt="Screenshot 2020-05-18 at 09 58 46" src="https://user-images.githubusercontent.com/841956/82188449-2e05ec80-98ee-11ea-9308-7c9c2b58941c.png">

Reply:
Note that the current time display is similar to how it's done in WordPress -- having a `am/pm` switch button if it's 12 hours format. There are various other display/UX options for the time which don't require a `am/pm` switch by the user, should we perhaps wait until UX for the custom calendar is ready first (#1391)?

Reply:
`am`/`pm` was a relatively minor question on my side. A bigger question was `mm/dd` vs `dd/mm`. It looks like WP is quite explicit about this?

Reply:
Ah, alright. The `mm/dd` vs `dd/mm` is not actually related to the calendar itself currently. We have a custom local button/field that has the display implemented based on WP settings, and then the calendar opens separately on a popup when clicking that, not influencing how the field is being displayed.

Currently, the date field is actually using only the "Time Format" value of the WordPress settings and not the "Date format" -- the date button/field display is implemented the same way as the date field is in the WordPress editor. If `am/pm` are used, `mm/dd` is used, otherwise `dd/mm`. ("Date Format" setting of WordPress is mainly used for displaying the post publish dates in the front-end instead.) We could definitely change that -- but this is not necessarily related to the custom calendar and could be handled in a separate issue -- we'd need to decide if to use the locale from WP settings then or from the browser.

Thoughts?

Reply:
Yes. Let's continue this way. If it's a problem, we'll have to revisit.